<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimersEditorWidget</class>
 <widget class="QWidget" name="TimersEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/48x48/timer.png</normaloff>:/icons/48x48/timer.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="editName"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelChannel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Channel:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QComboBox" name="editType">
     <item>
      <property name="text">
       <string>Once</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Daily</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weekly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weekdays</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelUrl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="editUrl">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QDateEdit" name="editDate">
     <property name="date">
      <date>
       <year>2012</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>dd. MM. yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QTimeEdit" name="editStartTime"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="labelDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="labelStartTime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Start time:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="labelEndTime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>End time:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QTimeEdit" name="editEndTime"/>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelDisabled">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Disabled:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QCheckBox" name="checkBoxDisabled"/>
   </item>
   <item row="11" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="editChannel">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/add.png</normaloff>:/icons/24x24/add.png</iconset>
   </property>
   <property name="text">
    <string>New Task</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/remove.png</normaloff>:/icons/24x24/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/save.png</normaloff>:/icons/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save and Close</string>
   </property>
   <property name="toolTip">
    <string>Save and Close</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/close.png</normaloff>:/icons/24x24/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
